packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -94,6 +95,15 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
        */
       path: String,
 
+      /**
+       * The path aliases delimited by a comma
+       */
+      pathAliases: {
+        type: String,
+        reflectToAttribute: true,
+        attribute: true,
```

Comment:
There is no need to set `attribute: true` as it doesn't make any sense for string properties.

Reply:
Removed.